qcodes/utils/update/qcodes_updater.py

```diff
@@ -0,0 +1,174 @@
+from typing import Tuple
+
+import datetime
+import git
+import json
+import os
+import shutil
+import subprocess
+import urllib.request
+import warnings
+import qcodes
+
+class DirtyTreeException(Exception):
+    pass
+
+
+def address_keeper() -> Tuple[str, str]:
+    # QCoDeS source path
+    src = os.sep.join(qcodes.__file__.split(os.sep)[:-2])
+    time_stamp = str(datetime.date.today()) # Current date.
+    dst = src + '_backup' + '_' + time_stamp
+    return src, dst
+
+
+def qcodes_is_editable() -> bool:
```

Comment:
Now that we have the same function in `installation_info`, we should probably use that one here to avoid code duplication.